webserver/grpc/src/main/java/io/helidon/webserver/grpc/GrpcProtocolHandler.java

```diff
@@ -347,4 +394,52 @@ public MethodDescriptor<REQ, RES> getMethodDescriptor() {
             }
         };
     }
+
+    /**
+     * Gets or creates gRPC server metrics for the method being invoked. Note that
+     * duration and size metrics are currently recorded only for successful calls,
+     * using the {@link #OK_TAG}. If a call fails, it will only increment the number
+     * of started calls, but not record any of the other metrics.
+     */
+    private void initMetrics() {
+        if (grpcConfig.enableMetrics()) {
+            MetricsFactory metricsFactory = MetricsFactory.getInstance();
+            MeterRegistry meterRegistry = Metrics.globalRegistry();
+
+            Tag grpcMethod = Tag.create("grpc.method", route.method().getFullMethodName());
+
+            Counter.Builder callStartedBuilder = metricsFactory.counterBuilder(
+                            "grpc.server.call.started")
+                            .scope(VENDOR);
+            callStartedBuilder.tags(List.of(grpcMethod));
+            Counter callStarted = meterRegistry.getOrCreate(callStartedBuilder);
```

Comment:
Pure bike shedding comment...

The code could also chain the `tags` invocation onto the end of the `scope` invocation:
```java
Counter.Builder callStartedBuilder = Counter.builder("graph.server.call.started")
    .scope(VENDOR)
    .tags(List.of(grpcMethod));
Counter callStarted = meterRegistry(callStartedBuilder);
```
and depending on stylistic preferences (!) and readability could even do without the builder local variable:
```java
Counter callStarted = meterRegistry.getOrCreate(Counter.builder("graph.server.call.started")
                .scope(VENDOR)
                .tags(List.of(grpcMethod)));
```
(Same note for all the registrations.)